mitodl/ocw-hugo-themes
# feat: Educator Page

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-hugo-themes/issues/307

#### What's this PR do?
Educator's page created which can be accessed from homepage, by clicking "Educators start here" 

#### How should this be manually tested?

- This PR is Deployed [here](https://ocw-hugo-themes-pr-309--ocw-next.netlify.app/educator/)
- Clicking "Educators Start Here" in homepage should redirect to this educator page.
- One long, scrolling page
- on-page nav should stay sticky, at top of viewport when user scrolls down 
- Clicking on item in on-page nav should smooth-scroll down to the appropriate section, not jump.
- Verify that all links work fine
- Should be a part of ocw-www theme
- Compare with design: https://invis.io/7Y12598HHF6E#/462148864_Educators_Start_Here
- Proofread content: https://docs.google.com/document/d/1DTD0cHXB_n3UH1HZwyD8adFOfrgxfY_zSzY5pTTyKyE
- As much as possible, styles should follow existing styles (e.g. fonts & colors), especially those added in About Us Page #303
- Test the responsiveness of the page for various screen sizes, including:
    - 1920×1080
    - 1366×768 
    - 768×1024 
    - 414×816 
    - 375×812
- Verify that the embedded video play
- Test the page on multiple browsers"
- Any other instructions, mentioned in relevant ticket

#### Screenshots (if appropriate)
Laptop screen: 

![educator page SS](https://user-images.githubusercontent.com/93309234/147901832-b8cb0520-986c-4488-97d8-ea5d3b525783.png)

Mobile screen:

![mobile SS](https://user-images.githubusercontent.com/93309234/147902471-61536ce9-3638-4fc6-b4e9-c7464db2c362.png)
